engine/glfw/lib/win32/win32_window.c

```diff
@@ -1099,7 +1099,8 @@ static ATOM registerWindowClass( void )
     wc.cbWndExtra    = 0;                             // No extra window data
     wc.hInstance     = _glfwLibrary.instance;         // Set instance
     wc.hCursor       = LoadCursor( NULL, IDC_ARROW ); // Load arrow pointer
-    wc.hbrBackground = NULL;                          // No background
+    // Change from the default white background to black to improve the startup white window when the graphics drive is initialization
+    wc.hbrBackground = (HBRUSH)GetStockObject(BLACK_BRUSH);
```

Comment:
Thanks for this. I agree it's an improvement.

Some questions/discussions though:
Q: Is black universally "better"?

Q: I would also have liked us to use the chosen background color from the game.
From the linked issue: "what would be best is if we had some control over what is displayed at this point "

Reply:
Adding @britzl to the discussion.

Reply:
Yeah, we'd like to use the graphics.clear_color from game.project, but for now maybe accept this PR and then fix it properly later?

Reply:
I'm investigating the proper solution now.

Reply:
Best practices should be when adaptive game starts with splash background color (if splash background color is customizable).

Because this is when the window for reinitialization windows occurs (when the window class is register), the game engine should not have been initialized correctly at this time, and it takes some trick to read the preset values of the engine.

So if you're not sure, black is a more professional and common background color in most games.

Reply:
An @britzl  says, the `graphics.clear_color` is color we want here, and we want to figure out a good way to propagate it down to the window initialization. At that stage, we _have_ read the project config settings, and have access to the color, so now it's a matter of figuring out how to pass it down to those places that need it.